packages/frontend-2/components/workspace/ProjectList.vue

```diff
@@ -1,5 +1,13 @@
 <template>
```

Comment:
When logged in as member the actions in the sidebar still show, however members dont have permissions to access these actions

<img width="1074" alt="Screenshot 2024-12-18 at 17 05 50" src="https://github.com/user-attachments/assets/f4d45990-5a59-4aee-96f5-fa3d3ab04117" />